cross/ffmpeg7/Makefile

```diff
@@ -246,8 +252,6 @@ CONFIGURE_ARGS += --enable-libmfx
 CONFIGURE_ARGS += --enable-vaapi
 endif
 
-include ../../mk/spksrc.cross-cc.mk
-
```

Comment:
If I recall correctly validation against `TC_GCC` can only occur once `cross-cc|cmake|*.mk` is called as otherwise value is undefined.